src/HH4b/corrections/data/ttbar_sfs/glopart-v2/ttbarcorr_2022.json

```diff
@@ -0,0 +1,427 @@
+                1.0
+              ],
+              "content": [
+                0.00893963098521231,
+                0.013456328708226372,
+                0.014842453906927117,
+                0.020324645358829215,
+                0.0260932215149524,
+                0.028881238260055453,
+                0.030831884376061547,
+                0.03276995349052447,
+                0.035837684070482836,
+                0.0476333243675354,
+                0.0661229771005823,
+                0.09213935123933614,
+                0.15636791433783928
+              ],
+              "flow": 1.0
+            }
+          },
+          {
+            "key": "stat_dn",
+            "value": {
+              "nodetype": "binning",
+              "input": "Xbb",
+              "edges": [
+                0.0,
+                0.009999999776482582,
+                0.10000000149011612,
+                0.20999999344348907,
+                0.3100000023841858,
+                0.4000000059604645,
+                0.5,
+                0.6000000238418579,
+                0.699999988079071,
+                0.800000011920929,
+                0.8700000047683716,
+                0.9200000166893005,
+                0.9599999785423279,
+                1.0
+              ],
+              "content": [
+                0.00893963098521231,
+                0.013456328708226372,
+                0.014842453906927117,
+                0.020324645358829215,
+                0.0260932215149524,
+                0.028881238260055453,
+                0.030831884376061547,
+                0.03276995349052447,
+                0.035837684070482836,
+                0.0476333243675354,
+                0.0661229771005823,
+                0.09213935123933614,
+                0.15636791433783928
+              ],
+              "flow": 1.0
+            }
+          }
+        ],
+        "default": {
+          "nodetype": "binning",
+          "input": "Xbb",
+          "edges": [
+            0.0,
+            0.009999999776482582,
+            0.10000000149011612,
+            0.20999999344348907,
+            0.3100000023841858,
+            0.4000000059604645,
+            0.5,
+            0.6000000238418579,
+            0.699999988079071,
+            0.800000011920929,
+            0.8700000047683716,
+            0.9200000166893005,
+            0.9599999785423279,
+            1.0
+          ],
+          "content": [
+            0.9229941917073031,
+            1.0002339635826891,
+            0.9423961159854606,
+            1.0529965294585804,
+            1.1108077265454117,
+            1.1426185913976297,
+            1.0896874613041834,
+            1.0808239118303224,
+            1.109110711240747,
+            1.137529689625193,
+            1.2236616738132018,
+            1.3498995001429814,
+            1.6204426230831177
+          ],
+          "flow": 1.0
+        }
+      }
+    }
+  ],
+  "compound_corrections": []
+}
```

Comment:
The new files are missing the `PTJJ` SF. We can take it from the old file, but if you redid the measurement, we should probably take it all from a consistent set.